index.bs

```diff
@@ -675,8 +675,9 @@ Tooling for Discussions and Publications</h4>
 			for producing its documentation and deliverables
 			or for official group discussions
 			<em class=rfc2119>should</em> be usable
-			without additional cost
+			(without additional cost)
 			by all who wish to participate,
```

Comment:
I don't think the "by all...participants" is necessary...

Reply:
No, it is very intentional. We are not scoping this rule to the entire population of the world (as we are with the deliverables) but rather to the people who are actually intending to participate. If a group of 5 people want to set up a meeting, they do not need to clear their preferred conferencing tool by a professional accessibility auditing organization, they just need to clear it with each other.

Reply:
This wording is an improvement over previous wording which did not call out the user needs of people with disabilities. Nevertheless I'm still not picturing a good scenario for this in practice. So, here's the practical part that I'd like to see if we can address. 

Some of the tools that the W3C Community uses have the potential to be especially useful for how we work; yet if they are not accessible, and if there is not a comparable and accessible alternative, then either the accessibility barriers become blockers to inclusive development of standards, or individuals with disabilities may have to initiate blocking usage of a tool. So there's a reluctance to raise this, and often tools don't get fixed for long periods of time.

Given that W3C is a large community, of potential interest to developers wanting to showcase their tools, I'd be interested in language that encourages a more deliberate effort, for a limited time only (for instance, one month maximum) during which W3C groups pro-actively reach out to tool owners and encourage them to fix their accessibility bugs, before removing a tool from the lineup of tools used by a group. 

It's possible that language in the Process Document doesn't need further tweaking, only in the Art of Consensus language that backs this up. But the hoped-for outcome would be an approach for accessibility of tools that gets us a greater variety of accessible tools, more quickly.

Reply:
> We are not scoping this rule to the entire population of the world (as we are with the deliverables) but rather to the people who are actually intending to participate.

Possibly reiterating or echoing @brewerj's point, the decision on whether or not to participate might be at least partly based on the accessibility of the tools being used. We need to find a path to enough proactivity in choosing accessible tools so that the 6th person who _would_ join that group of 5 in @fantasai 's example is not excluded from doing so by the 5 agreeing amongst themselves to using Inaccessible Meeting Tool (inc).

Reply:
I am hopeful that people (notably team) will be proactive and guide groups to accessible tools. The purpose of process rules is to give the team and others the formal standing to say "not OK", which we currently lack. There is always a struggle and a balance to choose between utility — and exploring the cutting edge — and i18n and accessibility, and I don't think we should be excessively prescriptive for fear that we'll end up being able to use nothing, and be miles away from using the best tools. Should is right here, I think.

Reply:
@dwsinger I understood this mini-thread to be about the text "by all who wish to participate," rather than "should".

Does this overlap with the CEPC, if we're worried about inclusion and what is "not OK"? Do we want/need to say something in both places?

Reply:
Sorry yes, but I think that scoping to the audience is also right. "This tool does not work with right-to-left text" could be an i18n problem – but if the group has no users who regularly work in RTL, should we really stop the WG from using it? I think not. We should report the issue to the toolmaker and try/hope to get it fixed before our first RTL user arrives, for sure. The problem is that if we restrict ourselves to perfect tools, we risk finding ourselves miles from the leading edge and current tooling. The purpose of the rule is so that when a user arrives who can't use something, we are obliged to try to fix it or change — if we can. We don't have control over all tools, alas. The rule says that that user has a legitimate request for action that cannot be ignored (which is a huge improvement over the status quo).